Changed 'code file' to 'score' file

DaveVoyles opened the pull request:
The iris tutorial docs calls it a *'score'* file, which I think makes more sense. When I saw *'code'* I wasn't sure if it needed my main.py file (ex: linear_reg.py) or the score.

[The iris tutorial clarified that.](https://docs.microsoft.com/en-us/azure/machine-learning/preview/tutorial-classifying-iris-part-3)

I'm still not sure if I need the model ID generated from the previous step, though. The iris tutorial says you do, but this does not. 

EX: "To create a manifest, use the following command and provide the model ID output from the previous step:"

```az ml manifest create --manifest-name <new manifest name> -f score_iris.py -r python -i <model ID> -s service_schema.json```

The ```-i <model ID>``` step/param is missing from these instructions, but is present in the iris tutorial. Not sure if needed.

azurecla commented:
Hi __@singhkay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kasing). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

squillace commented:
@tysonn Looks like we missed one. Thanks, Kay! #sign-off.

singhkay commented:
@squillace Found a few more occurences

squillace commented:
So did i. @singhkay can you close this? I'll submit complete fixes for our world.

singhkay commented:
Done

DaveVoyles committed:
Changed 'code file' to 'score' file

The iris tutorial docs call it a 'score' file, which I think makes more sense. When I saw 'code' I wasn't sure if it needed my main.py file (ex: linear_reg.py) or the score.

The iris tutorial clarified that: https://docs.microsoft.com/en-us/azure/machine-learning/preview/tutorial-classifying-iris-part-3

I'm still not sure if I need the model ID generated from the previous step, though. The iris tutorial says you do, but this does not. 

EX: "To create a manifest, use the following command and provide the model ID output from the previous step:"

az ml manifest create --manifest-name <new manifest name> -f score_iris.py -r python -i <model ID> -s service_schema.json

The -i <model ID> step/param is missing from these instructions, but is present in the iris tutorial. Not sure if needed.
